docs/changelog.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [v2.0.0](https://github.com/kryptokrauts/contraect-maven-plugin/releases/tag/v2.0.0)
+
+This release introduces type mapping support for the [æternity naming system (AENS)](https://aeternity.com/protocol/AENS.html) and ChainTTL as well as improvements, better error handling and known issues. This release is improved and adapted to the [v3.0.0](https://github.com/kryptokrauts/aepp-sdk-java/releases/tag/v3.0.0) of the [aepp-sdk-java](https://github.com/kryptokrauts/aepp-sdk-java).
+
+### Bugfixes
+- [#39](https://github.com/kryptokrauts/contraect-maven-plugin/issues/39) an NPE can occur if waiting for the tx leads to "Tx not mined"
+- [#63](https://github.com/kryptokrauts/contraect-maven-plugin/issues/63) handle explicit return type unit properly
+- [#66](https://github.com/kryptokrauts/contraect-maven-plugin/issues/66) getNextNonce should return BigInteger.ONE if account doesn't exist
+
+### New Features
+- [#2](https://github.com/kryptokrauts/contraect-maven-plugin/issues/2) check contract bytecode for given contractId 
+- [#62](https://github.com/kryptokrauts/contraect-maven-plugin/issues/62) introduced Chain.ttl type
+- [#65](https://github.com/kryptokrauts/contraect-maven-plugin/issues/65) Support for AENS type
+
+### General changes
+- [#40](https://github.com/kryptokrauts/contraect-maven-plugin/issues/40) handle known internal errors from dryRun call
+- [#49](https://github.com/kryptokrauts/contraect-maven-plugin/issues/49) better readable error message on errors within dryRun
+- [#52](https://github.com/kryptokrauts/contraect-maven-plugin/issues/52) provided getter and setter for contract address
+- [#58](https://github.com/kryptokrauts/contraect-maven-plugin/issues/58) remove fixed gas limit from dryRun and make configurable via maven config
```

Comment:
is the gaslimit really configurable right now? just want to get this confirmed. otherwise everybody might have a problem if the SDK default is not sufficient

Reply:
Yes the gaslimit is configurable via the param `dryRunDeployGas` within the maen config. The default ist 50000l

Reply:
Added parameter to https://github.com/kryptokrauts/contraect-maven-plugin/blob/docs/mkdocs/docs/use-the-plugin/plugin-configuration.md